aws_iam_account_password_policy.tf

```diff
@@ -4,40 +4,40 @@
 
 
 resource "aws_iam_account_password_policy" "passwordRequireMinLength" {
-  minimum_password_length        = 6
+  minimum_password_length = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireMinLength14" {
-  minimum_password_length        = 13
+  minimum_password_length = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireLowerCase" {
-  require_lowercase_characters   = false
-  minimum_password_length        = 14
+  require_lowercase_characters = true
+  minimum_password_length      = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireNumber" {
-  require_numbers                = false
-  minimum_password_length        = 14
+  require_numbers         = true
+  minimum_password_length = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireUpperCase" {
-  require_uppercase_characters   = false
-  minimum_password_length        = 14
+  require_uppercase_characters = true
+  minimum_password_length      = 14
 }
 
 resource "aws_iam_account_password_policy" "passwordRequireSymbol" {
-  require_symbols                = false
-  minimum_password_length        = 14
+  require_symbols         = true
+  minimum_password_length = 14
 }
```

Comment:
**Accurics remediation**
 
**[LOW]: Password policy must prevent reuse of previously used passwords**
 
**Vulnerability**
It is recommended that the password policy prevent the reuse of passwords.Preventing password reuse increases account resiliency against brute force login attempts
 
**Remediation**
Previous Passwords must not be resused to increase resiliency against brute force attack

source: accurics-